components/learningUnits/LearningUnit.js

```diff
@@ -0,0 +1,32 @@
+import {
+  Card,
+  CardContent,
+  Typography,
+  CardActionArea,
+  CardActions,
+} from "@mui/material";
+import LearningUnitCheckbox from "./LearningUnitCheckbox";
+
+function LearningUnit({ learningUnit, isPreviouslyCompleted }) {
+  const { id: learningUnitId, name } = learningUnit;
+
+  return (
+    <Card sx={{ maxWidth: 345, display: "flex", m: 2 }}>
+      <CardActionArea href={`/learningUnits/${learningUnitId}`}>
```

Comment:
I find that hard-coding the learning unit page path could be prone to errors in the future, because it's hard to detect that this route is present here. Maybe you could create a variable `learningUnitPagePath = "learningUnits"` and then use it href, this way is more explicit and easier to change in case the path change.

Reply:
Good idea, done